src/components/tasks/card/index.tsx

```diff
@@ -35,6 +35,8 @@ const Card: FC<Props> = ({
   const iconWidth = '25px';
 
   const cardClassNames = [classNames.card];
+  const dates = localEndsOn == undefined? " " : JSON.stringify(localEndsOn).slice(1, 11);
+  const [dateTime, setDateTime] = useState(dates);
```

Comment:
Why are multiple dates used for a single dateTime?

Reply:
All these data coming from "content" props.  we iterate this, then get these multiple values.
And this is fine because we are putting result of Array iteration.